### PR TITLE
add default files

### PR DESCRIPTION
**Pull request by classabbyamp**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Reporting Security Issues in MiaowWare Projects
+
+Contact [security@miaow.io](mailto:security@miaow.io)
```
Comment (0x5c) [inline SECURITY.md]: Add note about not publicly disclosing

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Contributing to MiaowWare Projects
+
+## Before You Start
+
+- Make sure there's an issue for the feature, bugfix, or other improvement you want to make.
+- Make sure it's something that the project maintainers want.
+  We can discuss it and assign the issue to you.
+- Make sure work isn't already being done on the issue.
+
+> Some contribution guidelines can vary project-to-project, so make sure to also read the `DEVELOPING.md` file in the project repository.
```
Comment (0x5c) [inline CONTRIBUTING.md]: Markdown quotes don't render visibly in github, they appear super light grey on white

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+If there are too many, your contribution may be squashed when merged.
+You may want to squash your commits locally yourself:
+
+```sh
+git reset --soft [commit before your changes]
+git commit
+```
+
+Make sure to document your code as you go, in both comments and external documentation (in `/dev-notes/`) as needed.
+`dev-notes` is especially important if you introduce a new json file format or to document some development process (like the command to crush the various images in the repository).
```
Comment (0x5c) [inline CONTRIBUTING.md]: Still references the qrm reposiotry, we need to make the dev-notes references generic

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+**Test your changes.**
```
Comment (0x5c) [inline CONTRIBUTING.md]: Make this a subheading

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+If your code doesn't work, it's not ready for merging.
+Make sure you not only test intended behaviour, but also edge cases and error cases.
+
+If you're making a user-facing change, put a quick summary in `CHANGELOG.md` under the `[Unreleased]` heading.
+Follow the [Keep a Changelog][1] format.
```
Comment (0x5c) [inline CONTRIBUTING.md:35]: Make this be under a subheading "Document user-facing changes".

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+## When You're Ready to Merge
+
+1. When you have finished working on your contribution, create a pull request from your fork's branch into the master branch of the project.
+1. Read through and complete the pull request template.
+   If the checklist is not complete, your contribution will not be merged.
```
Comment (0x5c) [inline CONTRIBUTING.md]: Do we not accept WIP PRs? We should make it clear that this applies to a PR *after* it leaves "WIP" status.

**Review by 0x5c — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -7,7 +7,7 @@
   We can discuss it and assign the issue to you.
 - Make sure work isn't already being done on the issue.
 
-> Some contribution guidelines can vary project-to-project, so make sure to also read the `DEVELOPING.md` file in the project repository.
+Some contribution guidelines can vary project-to-project, so make sure to also read the `DEVELOPING.md` file in the project repository.
```
Comment (0x5c) [inline CONTRIBUTING.md]: Add "if it exists"?

**Review by 0x5c — APPROVED** [review]
With comment
